README.md

```diff
@@ -28,8 +28,8 @@ the name uftrace stems from the combination of user and ftrace.
 
 It can record data from:
 - User space C/C++/Rust functions (by dynamically patching functions at runtime,
-  or with code compiled with with `-pg`, `-mfentry`, `-finstrument-functions`,
-  or using clang with `-fxray-instrument`)
+  or with code compiled with with `-pg`, `-finstrument-functions` or using
+  `-fpatchable-function-entry=N` with selective NOP patching at runtime)
```

Comment:
I think the last part would be "or using `-fpatchable-function-entry=N` for selective NOP patching)".

Reply:
Ack. I just updated it.